src/components/LayoutHeader/Header.js

```diff
@@ -44,6 +44,16 @@ const Header = ({location}: {location: Location}) => (
       },
     }}>
     <ContainerWrapper>
+      {typeof window === 'undefined'
```

Comment:
**suggestion:** I think it would make sense to move this to an API at Gatsby level instead of executing the script during React render: https://www.gatsbyjs.com/docs/custom-html/